manim/mobject/svg/svg_mobject.py

```diff
@@ -432,39 +432,55 @@ def handle_transforms(self, element, mobject):
             y = -self.attribute_to_float(element.getAttribute("y"))
             mobject.shift(x * RIGHT + y * UP)
 
-        transform = element.getAttribute("transform")
-        suffix = ")"
-
-        # Transform matrix
-        if transform.startswith("matrix(") and transform.endswith(suffix):
-            transform = transform[len("matrix(") : -len(suffix)]
-            transform = string_to_numbers(transform)
-            transform = np.array(transform).reshape([3, 2])
-            x = transform[2][0]
-            y = -transform[2][1]
-            matrix = np.identity(self.dim)
-            matrix[:2, :2] = transform[:2, :]
-            matrix[1] *= -1
-            matrix[:, 1] *= -1
-
-            for mob in mobject.family_members_with_points():
-                mob.points = np.dot(mob.points, matrix)
-            mobject.shift(x * RIGHT + y * UP)
-
-        elif transform.startswith("scale(") and transform.endswith(suffix):
-            transform = transform[len("scale(") : -len(suffix)]
-            scale_values = string_to_numbers(transform)
-            if len(scale_values) == 2:
-                scale_x, scale_y = scale_values
-                mobject.scale(np.array([scale_x, scale_y, 1]), about_point=ORIGIN)
-            elif len(scale_values) == 1:
-                scale = scale_values[0]
-                mobject.scale(np.array([scale, scale, 1]), about_point=ORIGIN)
-
-        elif transform.startswith("translate(") and transform.endswith(suffix):
-            transform = transform[len("translate(") : -len(suffix)]
-            x, y = string_to_numbers(transform)
-            mobject.shift(x * RIGHT + y * DOWN)
+        transform_attr_value = element.getAttribute("transform")
+
+        # parse the various transforms in the attribute value
+        transform_names = ["matrix", "translate", "scale", "rotate", "skewX", "skewY"]
+
+        # Borrowed/Inspired from:
+        # https://github.com/cjlano/svg/blob/3ea3384457c9780fa7d67837c9c5fd4ebc42cb3b/svg/svg.py#L75
+
+        # match any SVG transformation with its parameter (until final parenthese)
+        # [^)]*    == anything but a closing parenthese
+        # '|'.join == OR-list of SVG transformations
+        transform_regex = "|".join([x + r"[^)]*\)" for x in transform_names])
+        transforms = re.findall(transform_regex, transform_attr_value)
+
+        number_regex = r"[-+]?(?:\d+(?:\.\d*)?|\.\d+)(?:[eE][-+]?\d+)?"
```

Comment:
Not a blocking issue, but I wonder if this is similar to the `string_to_numbers` call

Reply:
for now, I would prefer to use the regex as I have taken it from the mentioned repo and have tested multiple SVGs through it (from `transforms` point of view). Maybe a follow-up to see if string_to_numbers is okay and/or modify the string_to_numbers to use this regex.